test/e2e/framework/pv_util.go

```diff
@@ -295,53 +364,72 @@ func CreatePVsPVCs(numpvs, numpvcs int, c clientset.Interface, ns string, pvConf
 
 	// create pvs and pvcs
 	for i = 0; i < pvsToCreate; i++ {
```

Comment:
those i's are not re-used, but initialized again and again. No reason not to use `i := 0` here.